README.md

```diff
@@ -19,10 +19,86 @@ build and deployment processes.
 
 [**Documentation is Available Online!**][3]
 
+# Connection Pool
+
+The NuoDB Node.js driver comes with a built in connection pool available.
+
+### Arguments
+
+**connectionConfig:** the configuration that will be used to create the connection in the pool, required argument.
+
+**minAvailable:** initial size of the pool and the number of connection the pool will aim to maintain, default of 10 is used if no argument is provided.
+
+**maxAge:** Amount of time from connection creation until it will age out, default of 300000ms (5 minutes) is used if no argument is provided.
+
+**checkTime:** how often the pool will run an internal liveliness check on free connections, default of 120000ms(2 minutes) is used if no argument is provided.
+
+**maxLimit:** hard cap on the amount of live connection the pool can maintain, default of 200 is used if no argument is provided. If 0, the pool will have no hard cap.
+
+**connectionRetryLimit:** amount of times a pool will attempt to create a connection, default of 5 is used if no argument is provided.
+
+**id:** optional argument to give the pool an id. As default the pool will be provided the “new Date().getTime()” at the time of its creation as its id.
```

Comment:
I don't know if it is possible to have more than one pool with the same id based on granularity.  I was more thinking we give it a one up number, based on some static value, the timestamp can remain for possible value on when a pool might have been created.